diff --git a/node_modules/@ionic/app-scripts/config/webpack.config.js b/node_modules/@ionic/app-scripts/config/webpack.config.js
index d152b2b..5a605cb 100644
--- a/node_modules/@ionic/app-scripts/config/webpack.config.js
+++ b/node_modules/@ionic/app-scripts/config/webpack.config.js
@@ -75,7 +75,7 @@ var devConfig = {
 
   resolve: {
     extensions: ['.ts', '.js', '.json'],
-    modules: [path.resolve('node_modules')]
+    modules: ['node_modules']
   },
 
   module: {
@@ -122,7 +122,7 @@ var prodConfig = {
 
   resolve: {
     extensions: ['.ts', '.js', '.json'],
-    modules: [path.resolve('node_modules')]
+    modules: ['node_modules']
   },
 
   module: {
