diff --git a/node_modules/@ionic/app-scripts/config/webpack.config.js b/node_modules/@ionic/app-scripts/config/webpack.config.js
index d152b2b..7b2f67f 100644
--- a/node_modules/@ionic/app-scripts/config/webpack.config.js
+++ b/node_modules/@ionic/app-scripts/config/webpack.config.js
@@ -23,6 +23,23 @@ var optimizedProdLoaders = [
   {
     test: /\.js$/,
     loader: [
+      {
+        loader: 'babel-loader',
+        options: {
+          presets: [
+            [
+              '@babel/preset-env',
+              {
+                'targets': {
+                  'android': '4.4',
+                  'ios': '9'
+                }
+              }
+            ]
+          ]
+        }
+      },
+
       {
         loader: process.env.IONIC_CACHE_LOADER
       },
@@ -75,11 +92,28 @@ var devConfig = {
 
   resolve: {
     extensions: ['.ts', '.js', '.json'],
-    modules: [path.resolve('node_modules')]
+    modules: ['node_modules']
   },
 
   module: {
     loaders: [
+      {
+        test: /\.js$/,
+        loader: 'babel-loader',
+        options: {
+          presets: [
+            [
+              '@babel/preset-env',
+              {
+                'targets': {
+                  'android': '4.4',
+                  'ios': '9'
+                }
+              }
+            ]
+          ]
+        }
+      },
       {
         test: /\.json$/,
         loader: 'json-loader'
@@ -122,7 +156,7 @@ var prodConfig = {
 
   resolve: {
     extensions: ['.ts', '.js', '.json'],
-    modules: [path.resolve('node_modules')]
+    modules: ['node_modules']
   },
 
   module: {
@@ -155,4 +189,3 @@ module.exports = {
   dev: devConfig,
   prod: prodConfig
 }
-
